app/models/downloader.rb

```diff
@@ -0,0 +1,21 @@
+class Downloader < ActiveRecord::Base
+  validates :name,
+    null: false,
+    presence: true,
+    format: { with: /(?<name>[A-Z][a-zA-Z]{2,})/},
```

Comment:
Name format might be too strict (requiring first letter to be a capital letter and no punctuation allowed).